README.md

```diff
@@ -1 +1,21 @@
-# hw01
\ No newline at end of file
+# hw01
+<h2> Michael Lynch </h2>
+<h3>Name</h3>
```

Comment:
Markdown language in itself is sufficient in rendering headings. I suspect that adding HTML language along with the markdown language is what prevented your changes from showing.